src/FSharp.UMX.fs

```diff
@@ -9,15 +9,19 @@ open System
 
 [<MeasureAnnotatedAbbreviation>] type bool<[<Measure>] 'm> = bool
 [<MeasureAnnotatedAbbreviation>] type byte<[<Measure>] 'm> = byte
+[<MeasureAnnotatedAbbreviation>] type uint16<[<Measure>] 'm> = uint16
 [<MeasureAnnotatedAbbreviation>] type uint64<[<Measure>] 'm> = uint64
 [<MeasureAnnotatedAbbreviation>] type Guid<[<Measure>] 'm> = Guid
+[<MeasureAnnotatedAbbreviation>] type char<[<Measure>] 'm> = char
 [<MeasureAnnotatedAbbreviation>] type string<[<Measure>] 'm> = string
 [<MeasureAnnotatedAbbreviation>] type TimeSpan<[<Measure>] 'm> = TimeSpan
 [<MeasureAnnotatedAbbreviation>] type DateTime<[<Measure>] 'm> = DateTime
 [<MeasureAnnotatedAbbreviation>] type DateTimeOffset<[<Measure>] 'm> = DateTimeOffset
 #if NET6_0_OR_GREATER
 [<MeasureAnnotatedAbbreviation>] type DateOnly<[<Measure>] 'm> = DateOnly
 [<MeasureAnnotatedAbbreviation>] type TimeOnly<[<Measure>] 'm> = TimeOnly
+#else
+[<MeasureAnnotatedAbbreviation>] type uint32<[<Measure>] 'm> = uint32
```

Comment:
It looks like the `uint` alias and the ability to support for UOM in `FSharp.Core` wasn't added until around F# 5. To work around this I think we need to use conditional compilation to add support for `uint32` prior to this point in time hence the `#else` in the preprocessor directive. 

I feel like this might not be the most accurate preprocessor directive to use though. So if anyone knows of a more F# language version specific one that we could use please let me know. I kinda wanted to use `FSHARP5_OR_GREATER` but couldn't find a reference for any commonly defined F# specific directives.